### PR TITLE
[QUICK FIX]: Update dispatcher connector get functions

### PR DESCRIPTION
**Pull request by aleccohan**
Update dispatcher connector "GET" functions to have the correct hostname.    
    
    
    ## Secure Coding Practices Checklist GitHub Link
    - https://github.com/RedHatInsights/secure-coding-checklist

    ## Secure Coding Checklist
    - [ ] Input Validation
    - [ ] Output Encoding
    - [ ] Authentication and Password Management
    - [ ] Session Management
    - [ ] Access Control
    - [ ] Cryptographic Practices
    - [ ] Error Handling and Logging
    - [ ] Data Protection
    - [ ] Communication Security
    - [ ] System Configuration
    - [ ] Database Security
    - [ ] File Management
    - [ ] Memory Management
    - [ ] General Coding Practices

### CODE REVIEW

File: src/connectors/dispatcher/impl.js [hunk 1]
```diff
@@ -50,8 +49,7 @@ module.exports = new class extends Connector {
     }
 
     async fetchPlaybookRuns (filter = null, fields = null, sort_by = null) {
-        const uri = new URI(config.platformHostname);
-        uri.path('/api/playbook-dispatcher/v1/runs');
+        const uri = this.buildUri(host, 'playbook-dispatcher', 'v1', 'runs');
```
Comment (Jason-RH) [inline src/connectors/dispatcher/impl.js:52]: I think you'll need a `/api` in the uri

Comment (aleccohan) [reply src/connectors/dispatcher/impl.js]: Ah!  Yup!  adding it now!

Comment (aleccohan) [reply src/connectors/dispatcher/impl.js:52]: @Jason-RH I stand corrected!  We actually add the /api prefex automatically so manually adding it here is not necessary.

Comment (Jason-RH) [reply src/connectors/dispatcher/impl.js:52]: Ah, my bad!

**Review by Jason-RH — APPROVED** [review]

**Pull request merged**